### DISCUSSION TITLE
[Feature] Show photo owner in shared album

### DISCUSSION THREAD

**Discussion by aamirazad** [immich-app/immich#11625] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The feature I am requesting is to write the name of who added the photo to shared album just like google photos does. 

Immich:
![image](https://github.com/user-attachments/assets/5d69ce7c-6226-45a5-97e5-3fc88e68a3e8)

Google photos:
![image](https://github.com/user-attachments/assets/f23adfbb-4c21-438d-b926-b73998a7a459)

It is possible to find out who added the photo by clicking on it and going to info and looking at shared by but this is cumbersome and I like google photo's way of doing it better. If you think the name's looks cluttered you can make it only show up when the image is hovered on. Sorry if this has already been discussed, I couldn't find anything related to the topic.

Thanks!

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#8779
